src/gap_to_julia.jl

```diff
@@ -15,6 +15,8 @@ An internal type of GAP.jl used for tracking conversion results in `gap_to_julia
 """
 const RecDict = IdDict{Any,Any}
```

Comment:
Thinking about this, can't this become
```suggestion
const RecDict = IdDict{Any,GAP.Obj}
```

Reply:
Then `RecDict` cannot be used for the GAP to Julia conversion.